plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchQueryBuilder.java

```diff
@@ -130,6 +130,10 @@ private static QueryBuilder buildTermQuery(BoolQueryBuilder queryBuilder, String
             }
```

Comment:
Please change commit message according to https://chris.beams.io/posts/git-commit/

Reply:
Done